openspp-project/openspp-registry
# 641 fix gender deletion

## **Why is this change needed?**
To fix the validation when deleting a gender.

## **How was the change implemented?**
Modified the `gender` field type to many2one and enable the restriction of `ondelete`.

## **New unit tests**
```
None
```

## **Unit tests executed by the author**
```
None
```

## **How to test manually**
- On new instance or clean instance: Install `spp_farmer_registry_demo` or `spp_mis_demo`.
- Generate sample datas.
- Try delete a Gender Types.
- Error should be raised.

## **Related links**
https://github.com/OpenSPP/openspp-modules/issues/641
